notebooks/122-quantizing-model-with-accuracy-control/122-yolov8-quantization-with-accuracy-control.ipynb

```diff
@@ -0,0 +1,332 @@
+{
```

Comment:
<p>looks like it makes sense to compare the original and quantized model,&nbsp;to show what become better (size maybe) and to show that the accuracy has not drop</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1234/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Accuracy and performance comparison were added.